### ISSUE TITLE
loss is nan

### ISSUE THREAD

**Issue by bjliuzp** [pytorch/vision#1631]
I run the official routine "TorchVision Object Detection Finetuning Tutorial". The data used is also provided by the routine, but the training process will encounter the following errors. I can confirm that the boxes information is correct. How can I solve this problem?
torch - 1.4.0.dev20191124
torchvision - 0.5.0.dev20191129
Epoch: [0]  [ 0/60]  eta: 0:02:14  lr: 0.000359  loss: 3.0239 (3.0239)  loss_classifier: 0.7344 (0.7344)  loss_box_reg: 0.0182 (0.0182)  loss_mask: 0.8779 (0.8779)  loss_objectness: 0.6966 (0.6966)  loss_rpn_box_reg: 0.6967 (0.6967)  time: 2.2364  data: 0.0803  max mem: 2038
Epoch: [0]  [10/60]  eta: 0:00:57  lr: 0.003745  loss: 3.0239 (2.9495)  loss_classifier: 0.6830 (0.6673)  loss_box_reg: 0.0000 (0.0068)  loss_mask: 0.7392 (0.7569)  loss_objectness: 0.6807 (0.6709)  loss_rpn_box_reg: 0.7882 (0.8477)  time: 1.1451  data: 0.0551  max mem: 3014
Epoch: [0]  [20/60]  eta: 0:00:43  lr: 0.007132  loss: 2.6232 (2.4223)  loss_classifier: 0.5024 (0.4540)  loss_box_reg: 0.0000 (0.0043)  loss_mask: 0.7269 (1.5023)  loss_objectness: 0.5958 (-0.4529)  loss_rpn_box_reg: 0.8376 (0.9146)  time: 1.0178  data: 0.0479  max mem: 3014
Loss is inf, stopping training
{'loss_classifier': tensor(6.5823e+33, device='cuda:0', grad_fn=<NllLossBackward>), 'loss_box_reg': tensor(7.5402e+32, device='cuda:0', grad_fn=<DivBackward0>), 'loss_mask': tensor(inf, device='cuda:0', grad_fn=<BinaryCrossEntropyWithLogitsBackward>), 'loss_objectness': tensor(-6.0204e+32, device='cuda:0',
       grad_fn=<BinaryCrossEntropyWithLogitsBackward>), 'loss_rpn_box_reg': tensor(3.7395e+32, device='cuda:0', grad_fn=<DivBackward0>)}

Process finished with exit code 1

**Comment by fmassa** [comment]
Hi,

Just to understand the problem you are facing, you run the finetuning tutorial with no modification whatsoever, and you get nan's, is that right?

**Comment by bjliuzp** [comment]
@fmassa Yes, at the beginning I encountered this problem by training my own data. Later, running the routine directly also had this problem.

**Comment by fmassa** [comment]
Can you try running the colab notebook from https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html , which uses PyTorch 1.3.1 and torchvision 0.4.2? This worked fine on my colab, I just tried it.

Now I should just double-check to see if this is due to some version mismatch between PyTorch torchvision (your nightlies are not from the same day, maybe you should try updating them?), or a bug somewhere that got introduced.

**Comment by datumbox** [comment]
This issue appears to be stale, so I'm closing it. If you think it should remain open, feel free to reopen it.
